src/Tools/ExternalAccess/Razor/RazorDocumentPropertiesServiceWrapper.cs

```diff
@@ -13,16 +13,9 @@ internal sealed class RazorDocumentPropertiesServiceWrapper : DocumentProperties
     {
         public RazorDocumentPropertiesServiceWrapper(IRazorDocumentPropertiesService razorDocumentPropertiesService)
         {
-            DesignTimeOnly = razorDocumentPropertiesService.DesignTimeOnly;
```

Comment:
We confirmed they were always passing true? I don't know how 'false' would have been meaningful but maybe there's some evil going on here I don't know of...

Reply:
We're not hardcoding any value for this, we're still reading from the `razorDocumentPropertiesService`, just doing it directly rather than through DocumentPropertiesService. Our call is here: https://github.com/dotnet/roslyn/pull/46703/files/94e9327e911ac016562354a8e3aac3b6b22a7fa2#diff-ad1e3ee4def8272812671dc111c57acbR45